package.json

```diff
@@ -20,5 +20,10 @@
   },
   "engines": {
     "node": ">=14"
+  },
+  "dependencies": {
+    "primeflex": "^3.2.1",
+    "primeicons": "^6.0.1",
+    "primereact": "^8.6.1"
```

Comment:
Is there a specific reason why we have these dependencies in the root?
We already have them in the example, so that should be enough I think?

```suggestion
```

Reply:
Hello MichaelDeBoey,
I added those dependencies in package.json because PrimeReact doesn't work without them when I run `npm run dev`
Could you teach me how do I run ESLint on my code, please?

Reply:
You should run `npm install` in the `primereact` folder

Reply:
do I have to change `primereact-basic` to `primereact`, remove those dependencies from package.json then run `npm install` in the primereact folder?

Reply:
That's indeed correct 👍

Please also revert changes to `yarn.lock`
```sh
git checkout main ./yarn.lock
```

Reply:
Done!